Backpack-SwiftUI/Tests/Text/BPKAttributedTextTests.swift

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+import SwiftUI
+@testable import Backpack_SwiftUI
+
+class BPKAttributedTextTests: XCTestCase {
+    func test_multipleItems() {
+        let items: [BPKAttributedText.Item] = [
+            .init("Hello Backpack ", style: .heading1),
+            .init("have an "),
+            .init("Amazing day! ", style: .heading3, color: .abisko),
+            .init("Working with multiple "),
+            .init("Font styles is pretty neat!", style: .heading3, color: .primaryColor)
+        ]
+        
+        assertSnapshot(BPKAttributedText(items: items)
+                        .frame(width: 300)
```

Comment:
do you still need to set the width if you are setting the intrinsic content size in the assertSnapshot method?

Reply:
We're setting the width to force ellipsis. The point of the test is to show the Text will not go multiline.
If we don't force a width it will be 'endlessly long' :D